app/packages/_components/list_flank/list_flank.coffee

```diff
@@ -15,16 +27,37 @@ Template.list_flank.events
     else
       data.selected.set itemId
 
+    _allItemsSelected = instance.allItemsSelected
+    if allItemsSelected(instance)
```

Comment:
And regarding the issue raised by the comment - would probably be good to resolve it with this PR.  Seems a little fragile as it is - though i'm not sure what would cause the order of the collections to change.